Shoko.Server/Providers/TMDB/TmdbMetadataService.cs

```diff
@@ -2031,6 +2031,38 @@ private void PurgeCompany(int companyId, bool removeImageFiles = true)
 
     #region People
 
+    public async Task RepairMissingPeople()
+    {
+        var missingIds = new HashSet<int>();
+        var (updateCount, skippedCount) = (0, 0);
```

Comment:
Didn't see it earlier, but please don't use tuple destruction to assign two new variables and instead assign them separately.